en/docs/administration/configure-custom-domain.md

```diff
@@ -0,0 +1,27 @@
+# Configure Custom Domain
+
+A custom domain is essential for proper branding, discoverability, and credibility of your site. Choreo allows you to configure a custom domain for your organization's developer portal.
+
+Follow the steps below to configure the custom domain for the developer portal:
```

Comment:
We decided to use title caps when referring to the Developer Portal too right? 
Might have to use title caps in all applicable occurrences.

Reply:
Here I am not referring to the Choreo developer portal, hence did not use capitalization.

Reply:
done